### PR TITLE
Support for disabling JavaScript at navigation

### PR DESCRIPTION
**Pull request by Timkor**

### CODE REVIEW

**Comment by Timkor** [comment]
I am sorry, I am not writing any tests.

File: packages/goto/src/index.js [hunk 1]
```diff
@@ -56,12 +56,16 @@ module.exports = ({ timeout, ...deviceOpts }) => {
...
     page,
-    { url, media, adblock = true, headers = {}, waitFor = 0, disableAnimations = true, ...args }
+    { url, media, adblock = true, headers = {}, waitFor = 0, disableAnimations = true, disableJavaScript = false, ...args }
   ) => {
     if (adblock) {
       await engine.enableBlockingInPage(page)
     }
 
+    if (disableJavaScript) {
+      await page.setJavaScriptEnabled(!disableJavaScript)
```
Comment (Kikobeats) [inline packages/goto/src/index.js]: haha just `true` there is enough 😛

**Comment by Kikobeats** [comment]
@Timkor no problem, it looks good.

Do you know any URL for testing this?

File: packages/goto/src/index.js (hunk 1 above)
Comment (Timkor) [reply packages/goto/src/index.js]: You are completely right... 😝

**Comment by Timkor** [comment]
@Kikobeats You could send some HTML string with a script, which alters the page, to browserless. And then check if the alternation did not succeed. I think that's the easiest solution for testing.

File: packages/goto/src/index.js (hunk 1 above)
Comment (Kikobeats) [reply packages/goto/src/index.js]: that's right!

**Comment by Timkor** [comment]
@Kikobeats Thank you very much.

**Pull request merged**